azure-pipelines-release.yml

```diff
@@ -0,0 +1,137 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
+
+trigger: none
+pr: none
+
+pool:
+  name: '1ES-Hosted-DurableTaskFramework'
+  demands:
+    - ImageOverride -equals MMS2022TLS
+
+steps:
+# Start by restoring all the dependencies.
+- task: DotNetCoreCLI@2
+  displayName: 'Restore nuget dependencies'
+  inputs:
+    command: restore
+    verbosityRestore: Minimal
+    projects: |
+      src/DurableTask.Netherite.AzureFunctions/DurableTask.Netherite.AzureFunctions.csproj
+      src/DurableTask.Netherite/DurableTask.Netherite.csproj
+
+# Build the filtered solution in release mode, specifying the continuous integration flag.
+- task: VSBuild@1
```

Comment:
similarly, this target (Netherite Azure Functions) should come after the next one (Netherite)